src/main/mainWindow.ts

```diff
@@ -470,6 +503,32 @@ export async function createWindows() {
                 mainWin!.maximize();
             }
         });
+
+        mainWin.on("focus", () => {
```

Comment:
this is a really bad way to do this. why not use menu items with accelerators like previously?

Reply:
Because it refused to work with the menu items after hours of trying. If anyone is able to get it working that way instead that would be super helpful

Reply:
it works very much the same. did you maybe forget to remove the default zoom handlers? they might take priority over your accelerators

Reply:
it works very much the same. did you maybe forget to remove the default zoom handlers? they might take priority over your accelerators